src/static/src/components/SettingEmail.js

```diff
@@ -14,11 +14,16 @@ const SettingsEmail = ({
   setSelectedModel,
   models,
 }) => {
+  localStorage.getItem("email");
```

Comment:
This does not seem to be used

Reply:
You're absolutely right, I missed this when I moved the code.

Reply:
Fixed!